CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   custom CPU templates.
 - Added support for the `virtio-rng` entropy device. The device is optional. A
   single device can be enabled per VM using the `/entropy` endpoint.
+- Added a new CLI option `--bitwise` for the strip feature of
```

Comment:
what could be a reason for users to opt out?

Reply:
The reason is only that some bits are grouped. For example, when a group of 4 bits is 0b1100 and 0b1010, the output is going to be 0bx10x and 0x01x. In this case, users need to look at the input file with the output file to know the 'x' bits.

thb, I guess having only the bitwise one is fine. I'd appreciate it if you could share your thought.

Reply:
Indeed. I don't think users want to think much about the usage of the tool, so it's better to keep it simple. Thank you for sharing your thought.

Actually what I implemented before publishing this PR was replace it with bit-wise one. I'll make this PR back.

Reply:
Changed the opt-in approach to the approach to only have the bitwise strip.